.cirrus.yml

```diff
@@ -109,15 +109,29 @@ task:
     - electrum/tests/regtest/run_bitcoind.sh
   electrumx_service_background_script:
     - electrum/tests/regtest/run_electrumx.sh
+  matrix:
+    - env:
+        Binaries: 'Tarball'
+      depends_on:
+        - "linter: Flake8 Mandatory"
+    - env:
+        Binaries: 'AppImage'
+      app_image_setup:
+        - apt-get -y install libfuse2
+        - rm run_electrum
+        - VERSION="$(git describe --tags --always | cut -c 1-9)"
```

Comment:
I am confused about this cut (and btw it looks fragile). Can you give an example of what is being piped into it?

Reply:
This is how the appimage is getting its name:
https://github.com/spesmilo/electrum/blob/33ac0b45917d1dfcd6d673eab2f7840173c84068/contrib/build-linux/appimage/make_appimage.sh#L26-L27

Reply:
The initial attempt using this didn't succeed for some unknown reason, but it's functioning properly now.

Reply:
Okay but why is the cut needed?
Note that the commit hash used by git describe is not guaranteed to have a constant length. As there are more commits in a repository, it tends to get longer and longer.

Reply:
It was returning a commit hash longer than 9 characters, that's why I used `cut.` I tested  using ` VERSION=$(git describe --tags --dirty --always)` and it works well.